### PR TITLE
修正後のコード

### PR DESCRIPTION
**Pull request by Kaneda-miraidenshi**
修正後のコードです
I fixed the size of the button '送信'  and the spaces of each div.
In terms of the button, I used some 'input' and when I designed other 'inputs', I didn't write the class and only wrote the class name.
Therefore, I added their class name and made the button not affect CSS of other 'input'.

### CODE REVIEW

**Comment by oladhari** [comment]
@Kaneda-miraidenshi 
you have conflict in the file index.html
could you rebase?

**Comment by Kaneda-miraidenshi** [comment]
now I fixed it.

File: myselfのコピー.css [hunk 1]
```diff
@@ -73,8 +75,11 @@
 }
 
 .main{
-    height:230vh;
-    background-color:#FDF5E6;;
+  
+    background-color:#FDF5E6;
+}
+.main ol,h3{
+    color:sienna;
```
Comment (oladhari) [inline myselfのコピー.css]: better to be consistent, and use one type of color system (hex or rgba or just names)

File: myselfのコピー.css [hunk 2]
```diff
@@ -185,21 +190,18 @@ span{
 }
 
 .fgn{
```
Comment (oladhari) [inline myselfのコピー.css]: better to use more clear class names, it is easier for you to remember where this class name is used in the code :)

File: myselfのコピー.css [hunk 3]
```diff
@@ -185,21 +190,18 @@ span{
...
-   
- 
     background-color: #FDF5E6 ;
     color:sienna;
```
Comment (oladhari) [inline myselfのコピー.css]: here too, instead of using sienna you can use the relative hex name

**Review by oladhari — APPROVED** [review]
LGTM
just some remarks about color and classnames, once you fix them if you do not mind you can merge 
very good work

**Pull request merged**